DevShell/python-builds.nix

```diff
@@ -0,0 +1,13 @@
+# { pkgs }:
+# with (import ./mach-nix-src.nix { inherit pkgs; });
+with (import ./mach-nix-src.nix);
+
+{
+  commitizen = mkPython {
+    python = nixpkgs.python38;
+    requirements = ''
+      commitizen
+      setuptools
+    '';
+  };
```

Comment:
We use commitizen to produce conventional commit messages and build a changelog thereof. openintegrationhub doesn't use conventional commits, so this is probably not desired.